es/CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ Cuando envíes una pull request, por favor, acompáñala de una descripción del
 
 Todas las contribuciones tienen que ser revisadas por alguien.
 
-Puede darse que tu contribución pueda ser implementada inmediatamente por una de las personas que mantienen la codebase. De todas formas, normalmente, una nueva pull request necesita de algunas mejoras antes de poder ser aceptada. Otros contribuyentes (o robots auxiliares) pueden tener feedback. Si este es el caso, las personas responsables del mantenimiento te ayudarán a mejorar tu documentación y código.
+Puede darse que tu contribución pueda ser implementada inmediatamente por una de las personas que mantienen el codebase. En todo caso, una nueva pull request generalmente necesita de algunas mejoras antes de poder ser aceptada. Otros contribuyentes (o robots auxiliares) pueden dar retroalimenación. Si este es el caso, las personas responsables del mantenimiento te ayudarán a mejorar tu documentación y código.
```

Comment:
- El codebase: This is a tough one but I believe I'm going to stick to my version.

The feminine article makes more sense to me because I translate "codebase" as "base de código" and "base" is feminine. I've found some documentation from José Ramón Sampedro, a former academic from the Royal Spanish Academy (of the Spanish language) in which he also supports that way of reasoning. You can find it here: http://www2.ati.es/gt//lengua-informatica/externos/sampedr1.html#angli1

- The rest of suggestions, I agree, I think yours are most accurate!